frontend/src/features/admin-form/create/logic/components/LogicContent/InactiveLogicBlock/InactiveLogicBlock.tsx

```diff
@@ -98,6 +98,7 @@ export const InactiveLogicBlock = ({
         borderColor="neutral.300"
         cursor={isPreventEdit ? 'not-allowed' : 'pointer'}
```

Comment:
```suggestion
        cursor={isPreventEdit ? 'not-allowed' : 'default'}
```

Reply:
cursor="auto" seems to work as well. Which one shall we go with, I can make the same changes for the workflow cursor

Reply:
Key difference:
auto: Lets the browser decide the cursor based on the element or its context.
default: Forces the cursor to be the default (usually the arrow) regardless of context.

Maybe we use default? Then we definitely know the appearance won't change

Reply:
We usually take the [initial value](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#formal_definition) to revert to the original value,

Reply:
@KenLSM @kevin9foong

Reply:
'Auto' it is, let me make that change.